pom.xml

```diff
@@ -85,47 +90,43 @@
             </plugin>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <version>${surefire.maven.plugin.version}</version>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.21.0</version>
+                <configuration>
+                    <forkCount>9</forkCount>
```

Comment:
do not use Forks.  Use threads.  Forks require significant CPU usage and are limited with how many they can use.

Reply:
Updated code fixes this